src/data/items/foodBartending.js

```diff
@@ -0,0 +1,355 @@
+export default {
```

Comment:
This is almost certainly going to need to be balanced at least a LITTLE. I have a spreadsheet available to anyone who wants it which has everything organized and easy to read.